controllers/user.js

```diff
@@ -43,8 +44,8 @@ async function verify(req, res) {
 
   if(unsafeBreachesForEmail) {
     unsafeBreachesForEmail.forEach((breach) => {
-      breach.BreachDate = new Date(breach.BreachDate).toLocaleString("en-US", {year: "numeric", month: "long", day: "numeric"});
-      breach.DataClasses = breach.DataClasses.join(", ");
+      breach.BreachDate = HBSHelpers.prettyDate(breach.BreachDate);
```

Comment:
Nit: use same helper below on L61.

Reply:
I have a bigger refactor incoming that consolidates all the helpers and pushes this code out of the controller completely.